tests/template_tests/filter_tests/test_js_json.py

```diff
@@ -0,0 +1,52 @@
+from django.test import SimpleTestCase
+from django.utils.functional import lazy
+from django.utils.html import json_tag
+
+from ..utils import setup
+
+
+class JsJsonTests(SimpleTestCase):
```

Comment:
Where does the name "test_js_json.py". I'd follow the naming of the tag "test_json_tag.py", although I wonder what "tag" in "json_tag" refers to?